first_hour_on_vwb/working_with_data_collections.ipynb

```diff
@@ -122,22 +124,25 @@
    },
    "source": [
     "### Create a new workspace\n",
+    "<a href=\"create-new-ws\"></a>\n",
     "\n",
     "In order to create a data collection, you must first create a new workspace. \n",
     "Run the cell below to create a widget, then populate the widget's input fields and click the button to create your new workspace.\n",
     "\n",
     "Widget input parameters include:\n",
-    "- `workspace_id`: Must be unique and consist only of lowercase letters, numbers and underscores. Provide a workspace ID that suggests something about the contents of the data collection you'd like to create and include the date of its creation, such as `<STUDY_NAME>_<YYMMDD>_dc_ws`. *You cannot change the workspace ID after workspace creation.*\n",
-    "- `description`: Must be a string. This description will stay with the workspace after it becomes a data collection. Before you convert a workspace to a data collection, you can update this value in the UI.\n",
-    "- `name`: This is the name displayed in Data Collection modal once the workspace is converted to a data collection, so the value should communicate the purpose of workspace (e.g. `<STUDY_NAME> Data Collection`). Before you convert a workspace to a data collection, you can update this value in the UI.\n",
+    "- `Workspace Name`: Must be a string. This value is displayed in the Data Collection modal once the workspace is converted to a data collection, so the value should communicate the intended purpose (e.g. `<STUDY_NAME> Data Collection`).<br> While the Verily Workbench UI and this widget require a workspace name to be provided, the CLI does not; if no workspace name is provided to the CLI, a UUID is generated instead. \n",
+    "- `Description`: Must be a string. This description will stay with the workspace after it becomes a data collection. Before you convert a workspace to a data collection, you can update this value in the UI.\n",
+    "- `Workspace ID`: Must be unique and consist only of lowercase letters, numbers and underscores. Provide a workspace ID that suggests something about the contents of the data collection you'd like to create and include the date of its creation, such as `<STUDY_NAME>_<YYMMDD>_dc_ws`. *You cannot change the workspace ID after workspace creation.* \n",
+    "- `Workbench Group Name`: Must be a Verily Workbench group in which you are a member or admin. Group policy constraints for a given workspace must be set at the time of workspace creation, and are necessary but insufficient for granting members of the group workspace access.\n",
```

Comment:
I don't see an input field in the widget for group

Reply:
I have removed this content now that I've confirmed with Eng that it's not possible to add a group policy constraint via the CLI, and therefore the group name is not a parameter for this widget.